src/components/WalletSelect.vue

```diff
@@ -139,7 +139,7 @@ export default {
       <div
         id="terms-container"
         class="my-2 w-[300px] flex flex-row text-left"
-        v-if="requireTermsAccepted"
+        v-if="requireTermsAccepted && !ozow"
```

Comment:
Ozow won't be setting the requireTermsAccepted flag, so strictly speaking you don't need this.